modules/gapi/test/render/ftp_render_test.cpp

```diff
@@ -0,0 +1,74 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2018 Intel Corporation
+
+
+#ifdef HAVE_FREETYPE
+
+#include <random>
+
+#include <opencv2/core/utils/configuration.private.hpp>
+
+#include "../test_precomp.hpp"
+#include "api/ftp_render.hpp"
+
+namespace opencv_test
+{
+    static
+    std::string getFontPath()
+    {
```

Comment:
Please remove extra indentation from namespaces

"test_precomp.hpp" must be the first include without any conditions (before any `#ifdef`)

Reply:
Fixed